src/Symfony/Component/OptionsResolver/Tests/OptionsResolver2Dot6Test.php

```diff
@@ -1103,6 +1103,56 @@ public function testFailIfCyclicDependencyBetweenNormalizerAndLazyOption()
         $this->resolver->resolve();
     }
 
+    public function testCatchedExceptionFromNormalizerDoesNotCrashOptionResolver()
```

Comment:
I could add an assert true at the end if you think it's relevant. The point of the test is to check that no exception is generated, which phpunit does assert by default.